client/my-sites/domains/components/mapping-instructions/index.jsx

```diff
@@ -14,7 +14,7 @@ import {
 	MAP_EXISTING_DOMAIN_UPDATE_A_RECORDS,
 	MAP_SUBDOMAIN_WITH_CNAME_RECORDS,
 } from 'calypso/lib/url/support';
-import { WPCOM_DEFAULT_NAMESERVERS } from 'calypso/state/domains/nameservers/constants';
+import { WPCOM_DEFAULT_NAMESERVERS } from 'calypso/my-sites/domains/domain-management/name-servers/constants';
```

Comment:
Or we might as well merge those with the constants in `lib/domains`: https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/domains/constants.js#L15